platforms/quorum/configuration/roles/helm_component/templates/constellation.tpl

```diff
@@ -49,9 +55,16 @@ spec:
 {% endfor %}
 {% endif %}
 {% if network.config.consensus == 'raft' %}
+{% if network.config.genesis | default('', true) | trim == '' %}
```

Comment:
change to bootnode as per previous comment

Reply:
I have kept the check on genesis because its the same for IBFT and RAFT. IBFT wont have the bootnode part, but will still have the genesis part, so its better to have same checks for both.

Reply:
Ok, then in the sample files, remove the genesis part in the main networkyaml, and add a new sample for add-org which has the genesis part. Otherwise users will use the main sample and then code will not work as there is genesis part in it (in case they put some bogus value in that)

Reply:
Sure. That's better